src/pool/srv_pool.c

```diff
@@ -4298,6 +4347,60 @@ pool_find_all_targets_by_addr(struct pool_map *map,
 	return rc;
 }
 
+static int
+pool_find_all_targets_without_flags_by_addr(struct pool_map *map,
+			      struct pool_target_addr_list *list,
+			      struct pool_target_id_list *tgt_list,
+			      struct pool_target_addr_list *inval_list_out, uint32_t flags)
+{
```

Comment:
hmm, this looks strange, the function named by without_flags, but the parameter add "flags"?

this looks duplicate with pool_find_all_targets_by_addr(), if you only want to ignore tgt_list in your case, can you just use NULL for tgt_list in your case.

Reply:
The purpose of this function is to get TGTS without a specific FLAG.